Makefile

```
@@ -27,3 +27,7 @@ test-cov: generate vet
 
 clean:
 	@rm -rf ./reports ./out
+
+fmt:
+	@gofumpt -w .
+	@goimports --local github.com/nestoca/joy,github.com/nestoca -w .
```

Would this be equivalent?

```suggestion
	@goimports --local github.com/nestoca -w .
```

---

Not exactly. Basically the former separates third party imports, joy imports, and nestoca imports.
The suggestion separates third party imports from nesto imports. Both are fine. One is coarser and one is more fine-grained.

---

I see, so they would get grouped in different groups (`nestoca/joy/*` vs `nestoca/*`)? If so, we currently don't make that distinction in our other projects (see [screenshot](https://nestoca.atlassian.net/wiki/spaces/NP/pages/1896906768/Go+Development+Environment#GoLand)), so we may want to keep it the same (my suggestion above).